repository-data/webfiles/src/main/resources/site/freemarker/nationalindicatorlibrary/indicator.ftl

```diff
@@ -31,8 +31,8 @@
                             <div class="media__icon media__icon--geographic-coverage"></div>
                             <dl class="media__body">
                                 <dt id="geographic-coverage"><@fmt.message key="headers.geographicCoverage"/></dt>
-                                <dd data-uipath="ps.indicator.geographic-coverage">
-                                    ${indicator.geographicCoverage}
+                                <dd data-uipath="ps.indicator.geographic-coverage">                                    
```

Comment:
whitespace :( (at the end)

Install plugin for your IDE http://editorconfig.org/#download - it will take care of it for you.

Reply:
Done and done